### ISSUE TITLE
Janus error: "Stream stop failed. Stream was removed by force."

### ISSUE THREAD

**Issue by njam** [cargomedia/cm#2256]
We sometimes get "Stream stop failed. Stream was removed by force." from cm-janus.
```
CM_Janus_StopStreamError: Stream stop failed. Stream was removed by force. in CM/Janus/Service.php on line 104
```

cm-janus' `/stopStream`:
```js
      serviceLocator.get('http-client').detach(stream.plugin)
        .then(function() {
          res.send({success: 'Stream stopped'});
          return next();
        })
        .catch(function(error) {
          serviceLocator.get('logger').warn('Detaching plugin failed', context.clone().extend({exception: error}));
          stream.plugin.removeStream()
            .then(function() {
              res.send({error: 'Stream stop failed. Stream was removed by force.'});
              next();
            })
            .catch(function() {
              res.send({error: 'Stream stop failed. Stream was not removed.'});
              next();
            });
```

@tomaszdurka @vogdb how should we handle this?
What is causing this?
Should this really be an error sent back to CM, or maybe it should be handled transparently in cm-janus?

**Comment by vogdb** [comment]
I guess that we can response from cm-janus as usual in that case. Just log it as warning.

**Comment by tomaszdurka** [comment]
Yes, I think cm should not care about it. I would just make sure that stream is really removed (remember the problem with non-deletable stream).

**Comment by tomaszdurka** [comment]
As discussed with @njam for *forced-removal*:
- in cm-janus we will log a warning and return non-error response on the API
- no adjustments in CM are needed

@vogdb can you open ticket in cm-janus and close this? Thanks!

**Comment by vogdb** [comment]
Will be fixed in cargomedia/cm-janus#278
